package.json

```diff
@@ -33,13 +33,11 @@
     "async": "^2.1.2",
     "colors": "^1.1.2",
```

Comment:
should we be removing colors too? I think, we are not using this also.

Reply:
Cool. Thanks for bringing to notice. Will update.